docs/js-web-tools/bootstrap/bootstrap.rst

```diff
@@ -0,0 +1,241 @@
+
+http://getbootstrap.com/examples/theme/ 
+
+More Themes
+^^^^^^^^^^^
+
+You can extend the standard Bootstrap Theme by overriding some or all of its CSS, adjusting the colors and styles to suit your own application's needs. If you'd like to start from a ready-made theme, there are several theme galleries available online that specialize in Bootstrap Themes, such as WrapBootstrap.com (which has a variety of commercial themes) and Bootswatch.com (which offers free themes).  Some of the paid templates available provide a great deal of functionality on top of the basic Bootstrap theme, such as rich support for administrative menus, and dashboards with rich charts and gauges.   An example of a popular paid template is Inspinia, currently for sale for $18, which includes an ASP.NET MVC5 template in addition to AngularJS and static HTML versions.  A sample screenshot is shown below.
+
+.. image:: _static/theme-inspinia.png
+
+If you're interested in building your own dashboard, you may wish to start from the free example available here: http://getbootstrap.com/examples/dashboard/. 
+
+Components
+^^^^^^^^^^
+
+In addition to those elements already discussed, Bootstrap includes support for a variety of built-in UI components. 
+
+Glyphicons
+----------
+
+Bootstrap includes icon sets from Glyphicons (http://glyphicons.com), with over 200 icons freely available for use within your Bootstrap-enabled web application. Here's just a small sample:
+
+.. image:: _static/theme-glyphicons.png
+
+Input Groups
+------------
+
+Input groups allow bundling of additional text or buttons with an input element, providing the user with a more intuitive experience:
+
+.. image:: _static/input-groups.png
+
+Breadcrumbs
+-----------
+
+Breadcrumbs are a common UI component used to show a user their recent history or depth within a site's navigation hierarchy. Add them easily by applying the "breadcrumb" class to any <ol> list element. Include built-in support for pagination by using the "pagination" class on a <ul> element within a <nav>. Add responsive embedded slideshows and video by using <iframe>, <embed>, <video>, or <object> elements, which Bootstrap will style automatically. Specify a particular aspect ratio by using specific classes like "embed-responsive-16by9".
+
+JavaScript Support
+^^^^^^^^^^^^^^^^^^
+
+Bootstrap's JavaScript library includes API support for the included components, allowing you to control their behavior programmatically within your application. In addition, bootstrap.js includes over a dozen custom jQuery plugins, providing additional features like transitions, modal dialogs, scroll detection (updating styles based on where the user has scrolled in the document), collapse behavior, carousels, and affixing menus to the window so they do not scroll off the screen. There's not sufficient room to cover all of the JavaScript add-ons built into Bootstrap – to learn more please visit http://getbootstrap.com/javascript/. 
+
```

Comment:
We should link to http://getbootstrap.com/css/ and http://getbootstrap.com/components/ and http://getbootstrap.com/javascript/ so users can discover other components easily